contents/docs/deployment/deploy-kubernetes.md

```diff
@@ -151,6 +151,13 @@ Alternatively, a YAML file that specifies the values for the above parameters ca
 $ helm install -f my-values.yaml my-release .
 ```
 
+## Updating your PostHog
+
+```bash
+$ helm repo update
+$ helm upgrade -f <config_yaml> <cluster-name> posthog/posthog
```

Comment:
is `cluster-name` a param to helm upgrade? I think it just defaults to use whatever cluster you have setup on kubectl

Reply:
Was following the docs [here](https://helm.sh/docs/helm/helm_upgrade/) which labels `helm upgrade [RELEASE] [CHART] [flags]` so it's release name more so but the argument seems to exist.